diff --git a/test/Vagrantfile.vbox.win.erb b/test/Vagrantfile.vbox.win.erb
--- a/test/Vagrantfile.vbox.win.erb
+++ b/test/Vagrantfile.vbox.win.erb
@@ -100,5 +100,10 @@
     C:/Progra~1/Git/bin/git config --system --unset credential.helper
     C:/Progra~1/Git/bin/git config --system credential.helper store
 SHELL
 
+  # configure gitlab.test
+  config.vm.provision "shell", inline: <<-SHELL
+    Add-Content -Path "$env:SystemRoot/System32/drivers/etc/hosts" -Value "`n10.0.2.2 gitlab.test" -Encoding ascii
+SHELL
+
 end
